kasa/smart/modules/energymodule.py

```diff
@@ -43,12 +43,12 @@ def __init__(self, device: "SmartDevice", module: str):
 
     def query(self) -> Dict:
         """Query to execute during the update cycle."""
-        return {
+        ret_val = {
```

Comment:
This should be renamed to something else, but we should probably decide what'd be the proper name for such constructs, we have now both `q` and `req` in the code base from the looks of it...

Reply:
Done